### DISCUSSION TITLE
Hacktoberfest and rust-libp2p

### DISCUSSION THREAD

**Discussion by umgefahren** [libp2p/rust-libp2p#2979] [General]
I authored many minor and one big pull request, that might me merged. It's not incredibly important to me, but would it be possible to add the `hacktoberfest-accepted` to all pull requests this month?

**Comment by thomaseizinger** [comment]
Is that a label we need to add? Happy to do that, please ping me on any PR that you want it on! :)

**Reply by umgefahren** [reply]
It would be a label to a PR or a Tag to the repository. I will ping you then.
